guides/using-git-secrets.md

```diff
@@ -0,0 +1,23 @@
+---
+category: Tools
+expires: 2019-01-31
+---
+# Using git-secrets
```

Comment:
I feel like this doesn't actually add any more value than what's in the standard. Can we kill this till we have a base config?

Reply:
Agreed, I'll leave this open as I'm going to have the base config written (this is the motivator to do it now) and then we'll have more content (and more to review) @sid234 I'll be in touch :-)

Reply:
I'd rather we don't push visibly-incomplete things to master, personally. Things that need *some* attention are fair enough, but this should just be in a separate branch.